imagehash_test.go

```diff
@@ -56,6 +56,18 @@ func TestNewImageHash(t *testing.T) {
 	}
 }
 
+func TestNil(t *testing.T) {
+	hash := NewImageHash(0, AHash)
+	dis, err := hash.Distance(nil)
+	if dis != -1 {
+		t.Errorf("Distance is expected as %d but got %d", -1, dis)
+	}
+	expectedError := errors.New("other should not be nil")
+	if err != nil && err.Error() != expectedError.Error() {
```

Comment:
Rather than comparing the `Error()` string of the error value, it seems better to declare the error value as a global variable and compare it to `errors.Is` (or `==`).

Reply:
Would you like provide concrete suggestions?